base/cvd/cuttlefish/host/commands/cvd/server_command/generic.cpp

```diff
@@ -235,6 +246,21 @@ std::vector<std::string> CvdGenericCommandHandler::CmdList() const {
   return subcmd_list;
 }
 
+Result<std::string> CvdGenericCommandHandler::SummaryHelp() const {
+  return kSummaryHelpText;
+}
+
+bool CvdGenericCommandHandler::ShouldInterceptHelp() const { return false; }
+
+Result<std::string> CvdGenericCommandHandler::DetailedHelp(
+    std::vector<std::string>& arguments) const {
+  std::string replacement = "<command>";
+  if (!arguments.empty()) {
+    replacement = arguments.front();
+  }
+  return fmt::format(kDetailedHelpText, replacement);
```

Comment:
Not a fan of the format string with the `{}` being 200 lines away from what is filled in. Can `kDetailedHelpText` be moved closer, or inlined into this call?

Reply:
Done